server/run.py

```diff
@@ -1,3 +1,3 @@
-from app import app
+from app.app import app
```

Comment:
i want to say this is not supported in python 3...?

Reply:
Was kind of curious why this was inconsistent with my experience, so I did a bit of experimentation: top-level script's path injects its directory into sys.path, but other modules/packages do not.